Notesnook.API/Services/UserService.cs

```diff
@@ -148,6 +148,14 @@ public async Task SetUserKeysAsync(string userId, UserKeys keys)
             {
                 userSettings.MonographPasswordsKey = keys.MonographPasswordsKey;
             }
+            if (keys.InboxKeys == null)
```

Comment:
This will set the user's InboxKeys to null when user sets his MonographsPasswordKey or AttachmentsKey.